contrib/generate-version-script.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
```

Comment:
I seem to recall we had a good reason for doing it the way we do, @hughsie do you remember?

Reply:
Ah I remember now!
https://blogs.gnome.org/mcatanzaro/2018/02/16/on-python-shebangs/

Reply:
A better solution is to instead make the call to `generate-version-script.py` use your intended python3 like we do here: https://github.com/fwupd/fwupd/blob/master/meson.build#L371